app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -465,7 +467,8 @@ private void handleItemEvents(final ChatViewHolder chatViewHolder, final int pos
                         break;
                     case SUSI_MESSAGE:
                         chatViewHolder.messageStar.setVisibility( (model.isImportant()) ? View.VISIBLE : View.GONE);
-                        chatViewHolder.chatTextView.setText(model.getContent());
+                        chatViewHolder.chatTextView.setText(Html.fromHtml(model.getContent()));
```

Comment:
I know it is parsed but when i checked it for anchor in map it is not working. It think it is converted to string thats why it is not working correctly? Can you check it?